### ISSUE TITLE
Does your implementation reproduce the original paper results?

### ISSUE THREAD

**Issue by Arsey** [rahulbhalley/Progressive-Growing-of-GANs#2]

**Comment by mbbrodie** [comment]
I second @Arsey's question. Can you post example result images on your README page?

**Comment by RahulBhalley** [comment]
@mbbrodie @Arsey Yeah. I would love to present samples data from PGGAN to README. 
Actually I am planning to update the code to include the recent _Optimal Transport_ theory-based training improvements. I will try to present the results in nearly 3 to 4 weeks.
